modules/imaging_uploader/test/TestPlan.md

```diff
@@ -109,3 +109,28 @@
 	archive, answer 'Yes'. Check that this reupload fails with error code 4 (and not 2). 
 	Related to Redmine#14093 and PR#3555.
 	[Manual Testing]
+20. First, set the config setting 'ImagingUploader auto launch' to 'No'. Upload an MRI archive that can successfully be
```

Comment:
```suggestion
20. Set the config setting 'ImagingUploader auto launch' to 'No'. Upload an MRI archive that can successfully be
```
Remove all these 'First'. (including point 19), You can't have multiple 'first'.

Reply:
OK